python/tests/unit/arcticdb/version_store/test_query_builder.py

```diff
@@ -277,3 +277,28 @@ def test_querybuilder_groupby_then_groupby(lmdb_version_store_tiny_segment):
     received.sort_index(inplace=True)
     expected = df.groupby("col1").agg({"col2": "sum", "col3": "mean"}).groupby("col2").agg({"col3": "mean"})
     assert_frame_equal(expected, received)
+
+
+def test_querybuilder_pickling():
+    """QueryBuilder must be pickleable with all possible clauses."""
+
+    # TODO: Also check that `PythonRowRangeClause` is pickleable once `QueryBuilder.row_range` is implemented.
+    q = QueryBuilder()
+    # PythonAggregationClause
```

Comment:
The comments are next to the wrong clauses

Reply:
Ah yes, I swapped lines.